deepchem/metalearning/tests/test_maml_reload_torch.py

```diff
@@ -0,0 +1,88 @@
+"""Test that MAML models can be reloaded."""
+
+import pytest
+import numpy as np
+import deepchem as dc
+
+try:
+    import torch
+    import torch.nn.functional as F
+    from deepchem.metalearning import TorchMetaLearner, TorchMAML
+    has_pytorch = True
+except:
+    has_pytorch = False
+
+
+class SineLearner(TorchMetaLearner):
```

Comment:
add this to a `try` block, TorchMetaLearner may not be imported always (might be a cause for the failures of the Jax Tests